pkg/artifactory/resource/configuration/resource_artifactory_ldap_setting_test.go

```diff
@@ -145,6 +149,9 @@ resource "artifactory_ldap_setting" "ldaptestemailattr" {
 	user_dn_pattern = "uid={0},ou=People"
 	email_attribute = "mail_attr"
 }`
+	var onOrAfterVersion7571 = func() (bool, error) {
```

Comment:
Why copy/paste this func? Why not just put this in `acctest.go` or something?

Reply:
This is the only way it could work. I'll look into re-using the example from the shared module.

Reply:
Yea, I didn't notice it's a closure.